.jvmopts

```diff
@@ -1,6 +1,6 @@
 -Dfile.encoding=UTF8
 -Xms1G
--Xmx5G
+-Xmx6G
```

Comment:
Is this required by the JS build ? I'm asking as if that's the case we might have a look at how those builds go when this https://github.com/typelevel/cats/pull/3546 is in. We might not need this increase anymore. What do you think ?

Reply:
>We might not need this increase anymore. What do you think ?

Indeed.
# 3546 is already merged so remove this commit.